data/mods/More_Survival_Tools/items.json

```diff
@@ -99,7 +99,20 @@
     "turns_per_charge": 600,
     "ammo": "tinder",
     "revert_to": "tinderbox",
-    "use_action": "TINDERBOX_LIT"
```

Comment:
Okay now that you've got a nice clean, updated branch please remove the iuse and related entries for TINDERBOX_LIT.